docs/Actions.md

```diff
@@ -110,9 +110,9 @@ Dispatched when an error should be cleared. Note that this also clears any "erro
 None.
 
 
-## CLEAR_ITEMS
+## CLEAR_CONTENTS
```

Comment:
Contents is the name of the array in the cart store holding both items and models, hence why it didn't occur to me how ambiguous this is.  I'll change it to CLEAR_CART_CONTENTS to avoid confusion.